advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx

```diff
@@ -0,0 +1,487 @@
+</p>
+
+> In this example, SIB Visions VisionX analyzes the orders.fmb file and shows all included windows/canvases for the selection. 
+> 
+> The following screen shot shows the orders.fmb file in Oracle Forms Builder.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration10.png">
+</p>
+
+9. After the .fmb files is selected, a Task List screen displays the Oracle Forms migration progress.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration11.png">
+</p>
+
+> After the migrated screen is created, it is displayed in Design Mode in SIB Visions VisionX, where the user can adjust it according to the requirements.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration12.png">
+</p>
+
+> The SIB Visions VisionX Oracle Forms Migration Extension can migrate FMBs semi-automatically. All user interface elements (Windows/Canvas/Items) and the Oracle Forms Persistence (Blocks) can be migrated automatically. In a manual step, the Oracle Forms application logic (PL/SQL) is compiled into the database. The remaining UI Logic is then created in SIB Visions VisionX visually or manually with Java Code.
+
+#### Add logic visually with actions in SIB Visions VisionX
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration13.png">
+</p>
+
+#### Modify or add any Java code using Eclipse
+
+1. Import the application in Eclipse by clicking shift+alt in the SIB Visions VisionX main view and clicking the icon next to the application.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration14.png">
+</p>
+
+2. Select an UI element in SIB Visions VisionX and click the purple Eclipse button.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration15.png">
+</p>
+
+3. Eclipse navigates to the relevant Java Code. 
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration16.png">
+</p>
+
+4. Change the “Order Information” label in Eclipse.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration17.png">
+</p>
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration18.png">
+</p>
+
+> If you change the code outside of SIB Visions VisionX in Eclipse, the changes are pushed back into the VisionX visual development environment in real time. This enables unlimited app development for Citizen Developers / Business Users and Pro Developers.
+
+5. Open the screen in the SIB Visions VisionX Live Preview and change some of the data. The data will be updated in the Oracle Database.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration19.png">
+</p>
+
+### Oracle database to EDB Postgres Advanced Server migration
+
+In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended Server using the EDB's Migration Toolkit.
```

Comment:
Looks like it got missed.  We need to update MTK in all places.  6 I think.  Should use full name.  Migration Toolkit.